Automattic/wp-calypso
# Jetpack Offer Reset: Jetpack Backup Real-time should upsell Scan

### Changes proposed in this Pull Request

- Jetpack Backup Real-time will now upsell Jetpack Scan.
- Drops every designation of Jetpack Scan _Daily_. From now on, there is no more Jetpack Scan Daily, just Jetpack Scan.

### Testing instructions

1. Run this PR with Offer Reset A/B Test enabled.
2. Go to the Plans page.
3. Select "Get Backup" to purchase 'Jetpack Backup'.
4. Select "Get Backup Realtime"
5. You should now see an upsell to add "Jetpack Scan" (See video below)

![chrome-capture (5)](https://user-images.githubusercontent.com/11078128/91888139-a34cc180-ec40-11ea-9de4-14e1645b1dcf.gif)

6. Next verify that "Jetpack Scan _Daily_" is now just "Jetpack Scan".

![Markup 2020-09-01 at 10 55 35](https://user-images.githubusercontent.com/11078128/91888991-f70bda80-ec41-11ea-8083-6db41d8552bc.png)
.
**Should be "Jetpack Scan"**, Like this:

![Markup 2020-09-01 at 10 57 15](https://user-images.githubusercontent.com/11078128/91889005-fa06cb00-ec41-11ea-96dc-54d75032114f.png)
